### PR TITLE
Added class AccountService

### PR DESCRIPTION
**Pull request by BlondGT**
i didn't get what you meant in tenth task - can you be more specific?!thanks

### CODE REVIEW

File: src/homework1/AccountService.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import java.time.Month;
+import java.util.*;
```
Comment (YeaaaaH) [inline src/homework1/AccountService.java]: don't use wildcard imports

File: src/homework1/AccountService.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import java.util.stream.Collectors;
+
+public class AccountService {
+
+    public static List<Account> over500Balance (List<Account> accounts){
```
Comment (YeaaaaH) [inline src/homework1/AccountService.java]: you can remove static modifier for all methods

**Review by YeaaaaH — APPROVED** [review]
good job

File: src/homework1/AccountService.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    }
+
+    public boolean youngerThan2000Year(List<Account> accounts){
+        return accounts.stream()
+                .allMatch(account -> account.getBirthday().getYear() < 2000);
+    }
+
+    public double sumAllMaleBalance(List<Account> accounts){
+        return accounts.stream()
+                .filter(account -> account.getGender().equals("Male"))
```
Comment (YeaaaaH) [inline src/homework1/AccountService.java]: Add constant variable for string "Male" or use ENUM Gender

Comment (BlondGT) [reply src/homework1/AccountService.java]: then we should add constant variable for other methods too! i was thinking about doing this but i followed your tasks

**Pull request merged**